services/QuillLMS/app/controllers/api/v1/rule_feedback_histories_controller.rb

```diff
@@ -3,7 +3,9 @@ def by_conjunction
         raise ArgumentError unless params.include?('activity_id') && params.include?('conjunction')
         report = RuleFeedbackHistory.generate_report(
             conjunction: params['conjunction'],
-            activity_id: params['activity_id']
+            activity_id: params['activity_id'],
+            start_date: params['start_date'],
+            end_date: params['end_date']
```

Comment:
You used `permit` on the other one of these, I think it's probably worth refactoring this controller to do the same (even though there's already clearly an existing pattern of not using `permit` that you were just following).  Same for the `rule_detail` action.

Reply:
@anathomical good call, just refactored!